pkg/syncer/fullsync.go

```diff
@@ -369,14 +381,19 @@ func fullSyncGetEntityMetadata(ctx context.Context, pvList []*v1.PersistentVolum
 				}
 			}
 			volumeToCnsEntityMetadataMap[volume.VolumeId.Id] = cnsMetadata
+			if len(volume.Metadata.ContainerClusterArray) == 1 && metadataSyncer.configInfo.Cfg.Global.ClusterID == volume.Metadata.ContainerClusterArray[0].ClusterId && metadataSyncer.configInfo.Cfg.Global.ClusterDistribution == volume.Metadata.ContainerClusterArray[0].ClusterDistribution {
```

Comment:
Any specific reason we're checking the values against `volume.Metadata.ContainerClusterArray[0]`, and not `volume.Metadata.ContainerCluster`? Can they be different in any case?

Reply:
`volume.Metadata.ContainerCluster` is deprecated after vSphere 6.7u3 and was replaced by `volume.Metadata.ContainerClusterArray`

Reply:
Got it. So if `volume.Metadata.ContainerClusterArray` has more than one entry, does that imply it's a file volume and used in multiple clusters? If that's the case, I'm assuming we're not setting cluster distribution for such volumes as part of this feature.